src/Task.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, MarkdownRenderer } from 'obsidian';
 
-import type { Moment } from 'moment';
+import moment, { Moment } from 'moment';
```

Comment:
Please revert this change. There is a reason that I use `window.moment` everywhere. I use the moment that comes with Obsidian instead of also bundling it again.